src/index.html

```diff
@@ -5,8 +5,389 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com/%22%3E%22%3E">
+    <link rel="preconnect" href="https://fonts.gstatic.com/%22%3E%22%3E">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__img">
+        <img
+          class="img_item"
+          src="images/logo.png"
+          alt="Moyo Logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_item__is-active is-active">Apple</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav-link"
+              data-qa="nav-hover"
+            >Ноутбуки и компьютеры</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Гаджеты</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Планшеты</a>
+          </li>
+
+          <li class="nav_item">
+            <a  href="#" class="nav-link">Фото</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__card-container card-container" data-qa="card">
+        <img
+          class="card-container__img"
+          src="images/imac.jpeg"
+          alt="Imac img"
+        >
+        <div class="card-container__title-block">
+          <h2 class="card-container__title">
+            Моноблок APPLE A1419 iMac <br>27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card-container__product-code">
+            Код товара: 195434
+          </p>
+        </div>
+        <div class="card-container__review-block">
+          <div class="card-container__stars-block stars-block">
```

Comment:
star-block should have a modifier with number of active stars, that should work for any number of them